versioned_docs/version-4.6/developers/operations-api/clustering.md

```diff
@@ -190,8 +190,8 @@ Bulk create/remove subscriptions for any number of remote nodes. Resets and repl
 
 _Operation is restricted to super_user roles only_
 
-- operation _(required)_ - must always be `configure_cluster`
-- connections _(required)_ - must be an object array with each object following the `add_node` schema.
+- `operation` _(required)_ - must always be `configure_cluster`
```

Comment:
Same as earlier. Missing a `]` on 170 and that should fix parsing

Reply:
oooo....broken code blocks. that's a fun one. I'll see what I can catch...